python/paddle/v2/framework/tests/test_gradient_checker.py

```diff
@@ -1,46 +0,0 @@
-import unittest
```

Comment:
Since GradientChecker are widely used now. It does not need to write an independent unit-test.